docs/bsp.md

```diff
@@ -1308,6 +1313,23 @@ object ScalaPlatform {
 }
 ```
 
+#### Scala Test Params
+
+`ScalaTestParams` is a basic data structure that contains scala-specific
+metadata for testing Scala targets. This metadata is embedded in the `data:
+Option[Json]` field of the `buildTarget/test` request.
+
+```scala
+trait ScalaTestParams {
+  /** The test classes to be run in this test execution.
+    * It is the result of `buildTarget/scalaTestClasses`. */
+  def testClasses: List[ScalaTestClassesItem]
```

Comment:
Does it make sense to have this be `Option[List[ScalaTestClassesItem]]`? To distinguish "run 0 test classes" and "run every test class"

Reply:
I think it doesn't make sense. If the client is sending a test request, it has to test something, otherwise why do the request in the first place?

Reply:
`TestParams.targets` includes the targets, I expect that to run all test classes defined in that target. Imagine that `ScalaTestParams.testOptions: List[String]` gets added down the road and I want to customize it, then I don't want to have to provide `testClasses: []`